README.md

```diff
@@ -703,7 +706,14 @@ If this variable is not exported and similar packages are installed outside your
 conda environment, your tar file may not contain all the dependencies required
 for an isolated Python environment.
 
-After creating the tar file from the conda environment, you need to tell Python
+Alternatively, Triton also supports unpacked Conda execution environments, given
+it  points to an activation script to setup the Conda environment. Usually the
+Conda activation script is located in:
+````$path_to_conda_pack/lib/python3.7/site-packages/conda_pack/scripts/posix/activate```
+This speeds up the server loading time for models.
+
+After creating the tar file from the conda environment or creating a conda
+environment with a custom activation script, you need to tell Python
```

Comment:
I do not think we need to force the user to us conda-pack and then unpack their environment for Triton to use it. They can simply copy the activation script over to the right path for Triton

Reply:
I think it is a bit hacky to ask the users to copy the activation script.

For me the activation script is in a different location: ./lib/python3.9/venv/scripts/common/activate

I'm a bit hesitant to ask the users to copy their activation script since it looks like it depends on conda version.

I think asking the users to use conda-unpack or having a utility that automatically looks for the activation script is a safer path.

Reply:
chatted offline with @Tabrizian. It seems like it is a bit restricting to require the user to have to use `conda-pack` and then `conda-unpack` for all conda environments, especially if they are trying to save time by using the environment in its unpacked form in the first place.

Will follow up with auto-searching for the activation script in a later PR. However must note there's a risk we will find the wrong activation script (the python venv activation script does not work for a conda environment, as an example).